guide.md

```diff
@@ -38,53 +47,80 @@ For a complete set of options related to discovery see the [shibboleth documenta
+
+## Display of Seamless Access Login Button
 
-## Displaying the Login Button
+### Including the component library
 
-Include the following HTML somewhere on your SPs login page:
+You will need to reference the Seamless Access component library before you can use it. For the Standard implementation, you will always reference through a CDN:
 
+``` html
+<head>
+<!-- Include the Seamless Access Sign in Button & Discovery Service -->
+<script src="https://service.seamlessaccess.org/thiss.js"></script>
+</head>
 ```
-<script src="https://use.thiss.io/thiss.js"/>
-<div id="login"/>
+
+### Rendering the button
+
+Include a `<div>` container that is identified with an id tag where you want the button to appear on your HTML page:
+
+``` html
+<body>
+  <!-- Location for the Seamless Access login button -->
+  <div id='putMyLoginButtonHere'></div>
+</body>
+```
+
+And to display the button into this container, invoke the DiscoveryComponent.render function as a handler for the onload event on the window. This action will cause the button to initialize as soon as the window has finished loading. (NOTE: while it is possible to initialize the button component earlier, the behavior can be unpredictable and is not recommended.)
+
+``` javascript
 <script>
 window.onload = function() {
-   thiss.DiscoveryComponent.render({
-      loginInitiatorURL: 'https://sp.example.com/Shibboleth.sso/Login?target=https://sp.example.com/',
-   }, '#login');
+  // Render the Seamless Access button
+  thiss.DiscoveryComponent.render({
+    loginInitiatorURL: 'https://sp.example.com/Shibboleth.sso/Login?target=https://sp.example.com/',
+  }, '#putMyLoginButtonHere');
 };
 </script>
 ```
 
-There are three parts to this:
+The `render` function in this code, takes two parameters:
 
- 1. Include the thiss.js component. It is important that you retrieve the component from the same domain as the thiss-js service is provided from.
- 2. Include a *div* with the id *login* - this can be anywhere in your markup where you want the button to appear and doesn't have to sit next to the script tags.
- 3. Invoke the DiscoveryComponent.render function as a handler for the onload event on the window. This will cause the button to initialize as soon as the window has finished loading. It is possible to initialize the component earlier but this may cause it to behave unpredictably - YMWV.
+1. The `loginInitiatorURL:` - See the next section, "Integration of Login Button with your SAML SP", for more details
+2. The location where to render the button - here you will use the id tag that you used for your html `<div>` container, formatted using css selector styling in quotes.
 
 ## Integrating the Login Button with your SAML SP
```

Comment:
It would be good to go over this part. I'm not sure that I fully understand how it works.